src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/index.spec.tsx

```diff
@@ -238,12 +317,20 @@ describe('<StudentLiveRegistration />', () => {
   });
 
   it('does not render if we fail to initialize current subscription for lti user', async () => {
-    mockGetLiveSessions.mockRejectedValue(undefined);
+    const deferred = new Deferred();
+    fetchMock.get('/api/livesessions/?limit=999', deferred.promise);
+
+    const queryClient = new QueryClient();
 
     const { container } = render(
-      wrapInIntlProvider(<StudentLiveRegistration />),
+      wrapInIntlProvider(
+        <QueryClientProvider client={queryClient}>
+          <StudentLiveRegistration />
+        </QueryClientProvider>,
+      ),
     );
 
     expect(container.childNodes.length).toEqual(0);
+    deferred.resolve(undefined);
```

Comment:
I don't think this test is testing what its comment says.
Here your are testing that when the query is loading, the form is not displayed, but this has nothing to do with an error.
You should reject the deferred to test it imo.